app/views/admin/secrets/index.html.erb

```diff
@@ -1,11 +1,10 @@
-
-  <% if !@vault_enabled %>
```

Comment:
I dislike unrelated changes but this was some whitespace indenting that sneaked its way into my PR.